### DISCUSSION TITLE
Is it possible to disable integrated log rotation?

### DISCUSSION THREAD

**Discussion by sergejostir** [drakkan/sftpgo#267] [Q&A] [answered]
I will be rotating logs externally in combination with sending SIGUSR1. Is it possible to disable sftpgo's rotation based on size and log backup handling? I know one option is to just put some absurdly high values for those options, but that kinda doesn't feel right.

**Comment by drakkan** [comment, marked as answer]
Hi,

we use lumberjack for rotating logs, I did a quick lock at its code, while it allows to disable log rotation based on [max age](https://github.com/natefinch/lumberjack/blob/v2.0/lumberjack.go#L336) the rotation based on size will always [happen](https://github.com/natefinch/lumberjack/blob/47ffae23317c5951a2a6267a069cf676edf53eb6/lumberjack.go#L446) so currently the only way is to set the size high enough. I could try to send a patch but it is unlikely it will be accepted, this feature was already asked by others in the past and the author does not like it

**Reply by drakkan** [reply]
It could be fixed in lumberjack v3, take a look [here](https://github.com/natefinch/lumberjack/issues/115). If this will be available upstream I'll add this option to SFTPGo too
